amplify/backend/function/expressServer/src/controllers/stockController.js

```diff
@@ -24,7 +24,7 @@ const getStockPrice = async (req, res, next) => {
   }
 };
 
-// @route   GET api/stocks
+// @route   GET api/stock
 // @desc    Get all stocks
 // @access  Private - add auth middleware to make it private
 const getAllStocks = async (req, res, next) => {
```

Comment:
@caolan-power Is this getAllStocks function supposed to be here?

Reply:
Nope and can't see it on my local machine so strange

Reply:
should be ok now